JS/Lesson19/gulcin/recursion.js

```diff
@@ -136,3 +238,21 @@ const person = {
 		country: 'USA',
 	},
 };
+
+function checkProp(obj, propToCheck) {
+	let entries = Object.entries(obj);
+	if(entries.length === 0 || entries[0].length === 0) {
+		return 'No property!';
```

Comment:
It's always better if a function returns result of the same type:
```suggestion
		return false;
```